tests/suites/host_test.function

```diff
@@ -520,8 +522,15 @@ int execute_tests( int argc , const char ** argv )
                     int dep_id = strtol( params[i], NULL, 10 );
                     if( dep_check( dep_id ) != DEPENDENCY_SUPPORTED )
                     {
-                        unmet_dependencies[unmet_dep_count] = dep_id;
-                        unmet_dep_count++;
+                        if( unmet_dep_count < UNMET_DEP_MAX )
```

Comment:
I'd prefer to use `ARRAY_LENGTH`, like in development. Please backport `ARRAY_LENGTH` — we don't systematically backport test suite improvements, but we should backport them at least when we see a use for them.

Reply:
Yes make sense, I've done it.